doc/development/tutorials/todo.rst

```diff
@@ -107,6 +107,20 @@ is just a "general" node.
    <http://docutils.sourceforge.net/docs/ref/doctree.html>`__ and :ref:`Sphinx
    <nodes>`.
 
+.. DANGER::
```

Comment:
"danger" is a bit surprising to me. This tutorial let new developers know to create a new python file for extensions. What about using "attention" instead?

Reply:
I'm not sure. To me, it feels more like a danger because it took me nearly half a year, on and off, to figure out how to complete what I wanted.
I understand that if the reader is following the tutorial blindly, they won't hit this issue, but it may still be very confusing and painful to debug to ones who don't do this for various reasons and has some initial knowledge resulting in a misleading perception of the problem.

Reply:
Yeah, not a fan of danger either. Important or even just info would be more than appropriate